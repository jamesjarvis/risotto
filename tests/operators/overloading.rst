func (i int) incr1() {
    i = i + 1
}

func (i &int) incr2() {
    i = i + 1
}

op (i &int) - (i2 int) {
    println("CALLED")
}

a := 1

println(a)

a.incr1()
println(a)

a.incr2()
println(a)

1 - 2
