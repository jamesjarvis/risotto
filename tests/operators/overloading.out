1
1
2
CALLED
